Source/version.cs

```diff
@@ -1,4 +1,4 @@
 using System.Reflection;
-// Version 3.5.0, year 2018+4 month 3 day 14 
-[assembly: AssemblyVersion("3.5.0.40314")]
-[assembly: AssemblyFileVersion("3.5.0.40314")]
+// Version 3.6.0, year 2018+4 month 5 day 10
```

Comment:
Just a heads up that we'll need to bump this up a day, since we won't be able to actually tag and release until the 11th :)

Reply:
Done